### PR TITLE
Page1

### PR DESCRIPTION
**Pull request by Phildallen**

### CODE REVIEW

File: indexabout.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Montserrat+Alternates&display=swap" rel="stylesheet">
+    <meta name="description" content="About me HTML file">
+    <meta name="author" content="Phil Allen">
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="icon" href="images/codeicon.png" sizes="192x192">
+    <link rel="shortcut icon" href="images/codeicon.png">
+    <link rel="apple-touch-icon" href="images/codeicon.png">
+    <!-- <script defer src="js/index.js"></script> -->
```
Comment (JamesMcGraaw) [inline indexabout.html]: Don't really need to keep the js tag in as we're not putting js in. Maybe?

File: indexcontact.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    <link rel="shortcut icon" href="images/codeicon.png">
+    <link rel="apple-touch-icon" href="images/codeicon.png">
+    <!-- <script defer src="js/index.js"></script> -->
+</head>
+<body>
+    <h1>Contact</h1>
+    <div class="container">
+        <div class="contactbuttons">
+            <div>
+                <a href="mailto:phildallen@proton.me">
```
Comment (JamesMcGraaw) [inline indexcontact.html]: public repo - do you want your actual email address online?

File: indexcontact.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                    <img src="images/email.png" alt="email">
+                </a>
+            </div>
+            <div>
+                <a href="tel:+447481920593">
```
Comment (JamesMcGraaw) [inline indexcontact.html]: same for phone no.

File: indexprojects.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    <link rel="icon" href="iimages/codeicon.png" sizes="192x192">
+    <link rel="shortcut icon" href="images/codeicon.png">
+    <link rel="apple-touch-icon" href="images/codeicon.png">
+    <!-- <script defer src="js/index.js"></script> -->
+</head>
+<body>
+    <h1>Projects page</h1>
+    <div class="container">
+        <div class="textbody">
+            <p>Lorem ipsum dolor sit amet. Id animi optio nam autem debitis in unde libero aut reiciendis sequi et facilis dolor aut Quis harum. Sed magni ipsum eum dolorem culpa vel maxime adipisci. Est unde alias et ratione magnam et sunt commodi et minima rerum. </p>
```
Comment (JamesMcGraaw) [inline indexprojects.html]: consider indenting close paragraph tag to be in line with opening tag.

File: css/styles.css [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+  
+}
+
+h1 {
+	font-size: 2.81rem;
+	text-align: center;
+	color: #ff6600;
+}
+
+p{
```
Comment (JamesMcGraaw) [inline css/styles.css]: space after p

File: css/styles.css [hunk 6]
```diff
@@ -0,0 +1,87 @@
+*, *::before, *::after {
+  box-sizing: border-box;
```
Comment (JamesMcGraaw) [inline css/styles.css]: consistency of spacing

File: indexhome.html [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="icon" href="images/codeicon.png" sizes="192x192">
+    <link rel="shortcut icon" href="images/codeicon.png">
+    <link rel="apple-touch-icon" href="images/codeicon.png">
+    <!-- <script defer src="js/index.js"></script> -->
+</head>
+<body>
+    <h1>Philip Allen</h1>
+	<h1>Junior Developer</h1>
+    <div class="navbuttonshome">
```
Comment (JamesMcGraaw) [inline indexhome.html]: consider nav class rather than just a div

File: indexabout.html [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+</head>
+<body>
+    <h1>About</h1>
+    <div class="container">
+        <div class="textbody">
```
Comment (JamesMcGraaw) [inline indexabout.html]: consider article rather than div

File: indexcontact.html [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="icon" href="images/codeicon.png" sizes="192x192">
+    <link rel="shortcut icon" href="images/codeicon.png">
+    <link rel="apple-touch-icon" href="images/codeicon.png">
+    <!-- <script defer src="js/index.js"></script> -->
+</head>
+<body>
+    <h1>Contact</h1>
+    <div class="container">
```
Comment (JamesMcGraaw) [inline indexcontact.html:20]: consider something not a div

File: indexprojects.html [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="icon" href="iimages/codeicon.png" sizes="192x192">
+    <link rel="shortcut icon" href="images/codeicon.png">
+    <link rel="apple-touch-icon" href="images/codeicon.png">
+    <!-- <script defer src="js/index.js"></script> -->
+</head>
+<body>
+    <h1>Projects page</h1>
+    <div class="container">
```
Comment (JamesMcGraaw) [inline indexprojects.html:19]: that same comment about divs

**Review by JamesMcGraaw — CHANGES_REQUESTED** [review]

File: indexabout.html (hunk 1 above)
Comment (Phildallen) [reply indexabout.html]: Removed

File: indexcontact.html (hunk 2 above)
Comment (Phildallen) [reply indexcontact.html]: Swapped out of fake adress

File: indexcontact.html (hunk 3 above)
Comment (Phildallen) [reply indexcontact.html]: Zeroed out

File: indexprojects.html (hunk 4 above)
Comment (Phildallen) [reply indexprojects.html]: done

File: css/styles.css (hunk 5 above)
Comment (Phildallen) [reply css/styles.css]: done

File: css/styles.css (hunk 6 above)
Comment (Phildallen) [reply css/styles.css]: corrected

File: indexhome.html (hunk 7 above)
Comment (Phildallen) [reply indexhome.html]: changed

File: indexabout.html (hunk 8 above)
Comment (Phildallen) [reply indexabout.html]: changed

File: indexcontact.html (hunk 9 above)
Comment (Phildallen) [reply indexcontact.html:20]: Not changed as I couldn't find anything suitable

File: indexprojects.html (hunk 10 above)
Comment (Phildallen) [reply indexprojects.html:19]: Not changed as I couldn't find anything suitable

**Review by JamesMcGraaw — APPROVED** [review]

**Review by nealioa — APPROVED** [review]

**Pull request merged**